eventconsumer/src/main/scala/com/gu/notifications/events/AthenaLambda.scala

```diff
@@ -66,7 +66,9 @@ class AthenaLambda {
     athenaAsyncClient.getQueryExecution(GetQueryExecutionRequest.builder()
       .queryExecutionId(id)
       .build()).thenComposeAsync((response: GetQueryExecutionResponse) => {
-      response.queryExecution().status().state() match {
+      val state = response.queryExecution().status().state()
+      logger.info(state.toString)
```

Comment:
Is it worth-adding something that you can search for in all cases here?

Reply:
Added id too as the id should be visible in the console later